packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/connector/useConnector.tsx

```diff
@@ -78,6 +94,7 @@ export const useConnector = (): UseConnectorValue => {
     if (params.nodeId) {
       updateNodeInternals(params.nodeId);
     }
+    setIsNewConnection(true);
```

Comment:
When we start a new connection or we start to reconnect an edge, onConnectStart is called.
We added `setIsNewConnection and isNewConnection` to know if we are starting a new connection or if we are reconnecting an existing edge.
The reason was that when we implemented [the feedback for edge creation](https://github.com/eclipse-sirius/sirius-web/pull/2331), the feedback for edge reconnexion was not supported.
So this line needs to be removed until we implement feedback for edge reconnexion.

Reply:
But this version of the PR *does* implement feedback for reconnection.
I commented yesterday that this was not yet supported, but that was for the version from yesterday.

Reply:
I was wrong: reconnection feedback only seemed to work "by chance" on the cases I tested. We actually do not have (and never have had) the backend information required for proper reconnection feedback.
I'll remove the half-baked and misleading attempt. Reconnection feedback will be handled later.